### DISCUSSION TITLE
希望能够   在动态页中  默认选中 “投稿视频” 栏 而不是 ”全部“栏

### DISCUSSION THREAD

**Discussion by Liyw979** [the1812/Bilibili-Evolved#2315] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/16919236/134657606-069a0c53-4e50-42a9-9e80-37013ce0e7ec.png)

**Comment by CKylinMC** [comment]
https://greasyfork.org/zh-CN/scripts/433045
